modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase

```diff
@@ -0,0 +1,166 @@
+@component-name = "portal-workflow"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property test.run.environment = "EE";
+	property testray.main.component.name = "Workflow Metrics";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Metrics");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-105495 - Verify that the user can transition tasks adding a comment to it"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test CanAddCommentToATransitionTask {
+
+		// TODO LRQA-67809 CanAddCommentToATransitionTask pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that users can bulk-transition pending tasks assigned to themselves from more than 1 instance via Workflow Metrics"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test CanBulkTransitionPendingTasksFromMultipleInstances {
+
+		// TODO LRQA-67806 CanBulkTransitionPendingTasksFromMultipleInstances pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that tasks that are not assigned to the current users can't be transitioned via bulk transition action"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test CanNotBulkTransitionTasksNotAssignedToCurrentUser {
+
+		// TODO LRQA-67814 CanNotBulkTransitionTasksNotAssignedToCurrentUser pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that there are loading cards in Select Steps to Transition and Choose Transition per Step"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test LoadingCardsAreDisplayed {
+
+		// TODO LRQA-67810 LoadingCardsAreDisplayed pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (Your connection has been unexpectedly lost. Select 'Done' to retry.) is displayed when the connection is lost before the user clicks on (Done)"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test MessageIsDisplayedWhenConnectionIsLost {
+
+		// TODO LRQA-67823 MessageIsDisplayedWhenConnectionIsLost pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (The selected steps have transitioned successfully) is displayed when several steps are transitioned at once"
```

Comment:
Please, change the description for `"LPS-108138 - Verify that the message -The selected steps have transitioned successfully- is displayed when several steps are transitioned at once"`